test/cvs-scm/run.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash -e
+#
+#  Quick functionality test for CVS
+#
+. ../test-lib.sh 2>/dev/null || { echo "Must run in script directory!" ; exit 1 ; }
+
+dir=$(mktemp -d)
```

Comment:
It's a good habit to insert a `trap 'rm -rf "$dir"' EXIT` directly afterwards. This will make sure that the temporary directory is removed regardless how the script exits.

Reply:
The ones I looked at didn't have that :)
But yes, that's a good idea.

Reply:
Oh, that's unfortunate. Looks like there is still room for improvement. :wink: